src/lib/parser/markdown.ts

```diff
@@ -21,6 +22,10 @@ enum SUPPORTED_ELEMENTS {
   title
 }
 
+enum SUPPORTED_METADATA {
```

Comment:
What if rather than using and maintaining an additional enum to check if something is supported or not we use the templates? If the template exists, then go ahead, otherwise skip that element.